### PR TITLE
Add aveva model

### PR DESCRIPTION
**Pull request by jmcook1186**
Adds Aveva model to case studies

Params:
- `pl` : power used by application
- `pb`: baseline power
- `time` : time to normalize to

returns:
`e-cpu` in kwh

via calculation:
`((pl - pb) * time)/1000`

### CODE REVIEW

**Comment by gnanakeethan** [comment]
@jmcook1186 Please push the ts files. I think you have committed the "js" files.

File: src/lib/case-studies/aveva-model.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import {IImpactModelInterface} from '../interfaces';
+import {KeyValuePair} from '../../types/boavizta';
+
+export class EAvevaModel implements IImpactModelInterface {
+  // Defined for compatibility. Not used in TEADS.
```
Comment (narekhovhannisyan) [inline src/lib/case-studies/aveva-model.ts:5]: be careful, this one is not teads

File: src/lib/case-studies/aveva-model.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  name: string | undefined;
+  /**
+   * Defined for compatibility. Not used here.
+   */
+  authenticate(authParams: object): void {
+    this.authParams = authParams;
+  }
+
+  /**
+   *  Configures the TEADS Plugin for IEF
```
Comment (narekhovhannisyan) [inline src/lib/case-studies/aveva-model.ts:17]: same here

File: src/lib/case-studies/aveva-model.ts [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
...KG9ic2VydmF0aW9uOiBLZXlWYWx1ZVBhaXIpIHtcbiAgICBpZiAoISgnbWVtLXV0aWwnIGluIG9ic2VydmF0aW9uKSB8fCAhKCd0aW1lc3RhbXAnIGluIG9ic2VydmF0aW9uKSkge1xuICAgICAgdGhyb3cgbmV3IEVycm9yKFxuICAgICAgICAnUmVxdWlyZWQgUGFyYW1ldGVycyBkdXJhdGlvbixjcHUtdXRpbCx0aW1lc3RhbXAgbm90IHByb3ZpZGVkIGZvciBvYnNlcnZhdGlvbidcbiAgICAgICk7XG4gICAgfVxuICAgIGlmICh0aGlzLm1lbW9yeUFsbG9jYXRpb24gPT09IDApIHtcbiAgICAgIHRocm93IG5ldyBFcnJvcihcbiAgICAgICAgJ1JlcXVpcmVkIFBhcmFtZXRlcjogbWVtLWFsbG9jIG5vdCBwcm92aWRlZCBpbiBjb25maWd1cmUnXG4gICAgICApO1xuICAgIH1cbiAgICBpZiAodGhpcy5tZW1vcnlFbmVyZ3kgPT09IDApIHtcbiAgICAgIHRocm93IG5ldyBFcnJvcihcbiAgICAgICAgJ1JlcXVpcmVkIFBhcmFtZXRlcjogbWVtLWVuZXJneSBub3QgcHJvdmlkZWQgaW4gY29uZmlndXJlJ1xuICAgICAgKTtcbiAgICB9XG5cbiAgICBjb25zdCBtZW1fYWxsb2MgPSB0aGlzLm1lbW9yeUFsbG9jYXRpb247XG4gICAgLy8gICAgY29udmVydCBjcHUgdXNhZ2UgdG8gcGVyY2VudGFnZVxuICAgIGNvbnN0IG1lbV91dGlsID0gb2JzZXJ2YXRpb25bJ21lbS11dGlsJ107XG4gICAgaWYgKG1lbV91dGlsIDwgMCB8fCBtZW1fdXRpbCA+IDEwMCkge1xuICAgICAgdGhyb3cgbmV3IEVycm9yKCdjcHUgdXNhZ2UgbXVzdCBiZSBiZXR3ZWVuIDAgYW5kIDEwMCcpO1xuICAgIH1cblxuICAgIGNvbnN0IG1lbV9lbmVyZ3kgPSB0aGlzLm1lbW9yeUVuZXJneTtcblxuICAgIHJldHVybiAobWVtX2FsbG9jICogKG1lbV91dGlsIC8gMTAwKSAqIG1lbV9lbmVyZ3kpIC8gMTAwMDtcbiAgfVxufVxuIl19
```
Comment (narekhovhannisyan) [inline src/lib/case-studies/aveva-model.ts:87]: what is this?

Comment (jmcook1186) [reply src/lib/case-studies/aveva-model.ts:87]: urgh - an accidental paste from the js file

Comment (jmcook1186) [reply src/lib/case-studies/aveva-model.ts:87]: fixing in new pr

**Pull request merged**